src/panels/developer-tools/event/developer-tools-event.ts

```diff
@@ -92,43 +91,31 @@ class HaPanelDevEvent extends LitElement {
   }
 
   private _parsedJSON() {
```

Comment:
This should be a memoized function that takes `this._eventData` as a param, so we dont load it every render but only when the `eventData` changes

Reply:
Not needed if we use `ha-yaml-editor`